### PR TITLE
Update project requirements / description

### PR DESCRIPTION
**Pull request by JArtisHyland**

### CODE REVIEW

File: FinalProject/README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Final Project
-Make them do the project.
+
+Per the description, students will put into practice the techniques we've built up over the semester.
```
Comment (josephmaxwellhyland) [inline FinalProject/README.md]: might want to link the description here!

File: FinalProject/StudentDesc.md
Comment (josephmaxwellhyland) [inline FinalProject/StudentDesc.md:1]: great!

**Review by josephmaxwellhyland — COMMENTED** [review]
good!!

File: FinalProject/README.md (hunk 1 above)
Comment (JArtisHyland) [reply FinalProject/README.md]: Fixed!

**Review by hylandtechoutreach — APPROVED** [review]
yayaya

**Pull request merged**